docs/advanced-ai/examples/understand-vector-databases.md

```diff
@@ -0,0 +1,51 @@
+---
+title: What are vector databases?
+description: Understand vector databases. Learn how n8n provides vector databases, along with the key components to work with them, including embeddings, retrievers, and document loaders.
+contentType: explanation
+---
+
+# What are vector databases?
+
+Vector databases store information as numbers:
+
+> A vector database is a type of database that stores data as high-dimensional vectors, which are mathematical representations of features or attributes. ([source](https://learn.microsoft.com/en-us/semantic-kernel/memories/vector-db){:target=_blank .external-link})
+
+This enables fast and accurate similarity searches. With a vector database, instead of using conventional database queries, you can search for relevant data based on semantic and contextual meaning.
+
+## A very simplified example
+
+A vector database could store the sentence "n8n is a source-available automation tool that you can self-host", but instead of storing it as text, the vector database stores numbers that represent its features. This doesn't mean turning each letter in the sentence into a number. Instead, the vectors in the vector database describe the sentence. 
+
+Suppose that in a vector store `1` represents `automation tool`, `2` represents `source available`, and `3` represents `can be self-hosted`. You could end up with the following vectors:
+
+| Sentence | Vector |
+| -------- | ------ |
+| n8n is a source-available automation tool that you can self-host | 123 |
+| Zapier is an automation tool | 1 |
+| Make is an automation tool | 1 |
+| Confluence is a wiki tool that you can self-host | 3 |
```

Comment:
Nitpicking but the vector would always be an array of fixed dimensions of values between 0 and 1 so more like `[0.123, 0.111, 0.03]`